packages/@tinacms/toolkit/src/packages/fields/plugins/ImageFieldPlugin.tsx

```diff
@@ -93,11 +92,9 @@ export const ImageField = wrapFieldsWithMeta<InputProps, ImageProps>(
             ? // @ts-ignore
               cms.media.store.parse(media)
             : media
-        props.input.onChange('')
```

Comment:
I'm not 100% what this was doing, but what was occurring on the dnd upload / set is the `props.input.onChange(parsedValue)` wasn't making it through to values, just and empty string.

I removed this and everything seemed to be working smoothly.

Happy to bring it back if it's crucial.

Reply:
from what I recall, uploading an image with the same filename (overwriting) was not triggering the `onChange` in some cases. I think this was from before the introduction of the media manager, and the fact that we now send objects through instead of strings may have changed this

Reply:
I'm gonna hold up merging here because I want to test out of this `onChange` is still needed or not

Reply:
So, looks as though `props.input.onChange('')` does indeed prevent the parsedValue from being passed to the form. Essentially meaning this could be a whole rabbit hole.

Wondering if there is some kind of basic check we could throw in the interim...

e.g. 
```
  if (typeof media store ==== Cloudinary) => onChange(parsedValue)
  else => onChange('') .... onChange(parsedValue)
```

Reply:
That's not really feasible, unfortunately. I'm fine with eating this regression until we can find a better way around it